frontend/debian/cuttlefish-orchestration.cuttlefish-cvdserver-bootstrapper.init

```diff
@@ -0,0 +1,110 @@
+#!/bin/sh
+#
+### BEGIN INIT INFO
+# Provides: cuttlefish-host-orchestrator
+# Required-Start: $network $remote_fs
+# Required-Stop: $network $remote_fs
+# Default-Start: 2 3 4 5
+# Default-Stop: 0 1 6
+# Short-Description: cuttlefish-cvdserver-bootstrapper service
+# Description: The cuttlefish-cvdserver-bootstrapper service is a companion service
+#              for the Host Orchestrator Services used to download and launch
+#              AOSP cvd server under a different user.
+### END INIT INFO
+#
+# Copyright (C) 2022 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Make sure calls to this script get redirected to systemctl when
+# using systemd
+
+. /lib/lsb/init-functions
+
+if [ -f /etc/default/cuttlefish-cvdserver-bootstrapper ]; then
+    . /etc/default/cuttlefish-cvdserver-bootstrapper
+fi
+
+http_port=${http_port:-16993}
+
+RUN_DIR="/run/cuttlefish-cvdserver-bootstrapper"
+DAEMON="/usr/lib/cuttlefish-common/bin/cvdserver-bootstrapper"
+PIDFILE="${RUN_DIR}"/cvdserver-bootstrapper.pid
+
+start() {
+  mkdir -p "${RUN_DIR}"
+  chown _cvd-executor:cvdnetwork "${RUN_DIR}"
+  chmod 775 "${RUN_DIR}"
+
+  mkdir -p  "${download_dir}"
+  chown _cvd-executor:cvdnetwork "${download_dir}"
+
+  HTTP_PORT="${http_port}" \
+  start-stop-daemon --start \
+    --pidfile "${PIDFILE}" \
+    --chuid _cvd-executor:cvdnetwork \
+    --background --no-close \
+    --make-pidfile \
+    --exec "${DAEMON}"
+}
+
+stop() {
+  start-stop-daemon --stop \
+    --pidfile "${PIDFILE}" \
+    --remove-pidfile \
+    --exec "${DAEMON}"
+}
+
+status() {
+  # Return
+  #   0 if daemon is running
+  #   1 if daemon is dead and pid file exists
+  #   3 if daemon is not running
+  #   4 if daemon status is unknown
+  start-stop-daemon --start --quiet --pidfile "${PIDFILE}" --exec ${DAEMON} --test > /dev/null
```

Comment:
`--start` will start the daemon if it's not running. `--status` seems like a more appropriate option for this purpose.

Reply:
`--test` should also be dropped, it only prints information which is immediately discarded in `/dev/null`

Reply:
`--test` was added here so no action is taken, so `--start` won't start the daemon if it's not running. It seems this combination was used for simulating `--status` which was added in version 1.16.1.

started using `--status`

Reply:
oh! is that version available in ubuntu 18.04? Maybe we should just leave it with `--start --test` to avoid breaking anything.